src/descriptor/satisfied_constraints.rs

```diff
@@ -25,6 +25,9 @@ use {BitcoinSig, ToPublicKey};
 /// Detailed Error type for Interpreter
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub enum Error {
+    /// An uncompressed public key was encountered in a context where it is
+    /// disallowed (e.g. in a Segwit script or p2wpkh output)
+    UncompressedPubkey,
```

Comment:
Shouldn't we provide an information on which key is uncompressed?

Reply:
This commit is part of #130